File: 1066.py
a=[]
b=0
c=0
d=0
e=0
for i in range(1,6):
    a.append(int(input()))
for i in a:
    if i%2==0:
        b=b+1
    if i%2==1 or i%2==-1:
        c=c+1
    if i>0:
        d=d+1
    if i<0:
        e=e+1
print(b,"valor(es) par(es)")
print(c,"valor(es) impar(es)")
print(d,"valor(es) positivo(s)")
print(e,"valor(es) negativo(s)")

File: 1066.cpp
#include<iostream>
using namespace std;
int main()
{
    int a,i,b=0,c=0,d=0,e=0;
    for(i=0; i<5; i++)
    {
        cin>>a;

        if(a%2==0)
        {
            b++;
        }
       if(a%2==1 || a%2==-1)
        {
            c++;
        }
        if(a>0)
        {
            d++;

        }
        if(a<0)
        {
            e++;

        }

    }
    cout<<b<<" valor(es) par(es)"<<endl;
    cout<<c<<" valor(es) impar(es)"<<endl;
    cout<<d<<" valor(es) positivo(s)"<<endl;
    cout<<e<<" valor(es) negativo(s)"<<endl;
    return 0;
}
